templates/components/itilobject/timeline/timeline_item_header_badges.html.twig

```diff
@@ -31,14 +31,20 @@
  # ---------------------------------------------------------------------
  #}
 
-<span class="badge text-wrap d-none d-md-block">
+{% set timeline_display_date = session('glpitimeline_date_format') %}
+
+<span class="badge user-select-auto text-wrap d-none d-md-block">
    {% set date_span %}
       <span title="{{ date_creation|formatted_datetime }}"
             data-bs-toggle="tooltip" data-bs-placement="bottom">
```

Comment:
The tooltip is only relevant when relative dates are displayed.

Reply:
done